ui/src/chat/useChatStore.ts

```diff
@@ -12,6 +12,7 @@ export interface ChatInfo {
     brief: ChatBrief; // lags behind actual brief, only gets update if unread
   };
   dialogs: Record<string, Record<string, boolean>>;
+  hovering: Record<string, boolean>;
```

Comment:
I think you just want this to be a string, because we shouldn't ever have more than one message hovered. As is right now you can get multiple hovers.